doc/source/user_guide/meshing_workflow/meshing_workflows.rst

```diff
@@ -1,9 +1,9 @@
 .. _ref_user_guide_meshing_workflows:
 
-Use meshing workflows
-=====================
-Fluent meshing guided workflows are designed to make it easier to generate a
-volume mesh starting from CAD geometries. You can use PyFluent to access the guided workflows.
+Classic meshing workflow
+========================
+Provides sample usage of ``watertight`` and ``fault-tolerant`` meshing by using
```

Comment:
This is a fragment and we don't need to say this here. A slight adjustment to the existing text was fine. Simply, the final sentence could read "You can use PyFluent to access the classic meshing workflows which align with the journaled syntax."